src/comps/blogList.tsx

```diff
@@ -0,0 +1,20 @@
+function BlogList(props) {
+  const { blogsList } = props;
+  const { blogTitle } = props;
+
+  return (
+    <div className="blogList">
+      <h2> {blogTitle} </h2>
+      {blogsList.map((blog) => (
```

Comment:
it seems you haven't used typescript because you didn't give types so you need to add typescript to your code